### PR TITLE
[cli] Add `crons` to build output

### PR DESCRIPTION
**Pull request by AndyBitz**
- [x] Updates `build-utils` to define the `crons` type
- [x] Updates `vercel build` to consider the `cron` property and output the `crons` config
- [x] Updates  `next` to consider the crons manifest file
- [x] Updates `node` to consider the `cron` property
- [x] Updates `static-config` to consider the `cron` property
- [x] Add tests

### CODE REVIEW

**Comment by vvo** [comment]
This looks good, as you said we're missing:
- tests
- documentation of the new `crons` and `cron` fields to https://vercel.com/docs/build-output-api/v3

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ importers:
       '@types/node': 14.18.33
       '@typescript-eslint/eslint-plugin': 5.21.0_5yvnzvdwzksrnum37j3gumwy4y
       '@typescript-eslint/parser': 5.21.0_eslint@8.14.0
-      '@vercel/build-utils': link:packages/build-utils
+      '@vercel/build-utils': 6.0.0
```
Comment (styfle) [inline pnpm-lock.yaml]: Why is the lockfile changing? Did use the correct version of pnpm?

https://github.com/vercel/vercel/blob/08cdfa2a057e835d13957933b8ec12a19e2031b4/package.json#L6

Comment (AndyBitz) [reply pnpm-lock.yaml]: Oh, definitely not expected, will revert

**Comment by AndyBitz** [comment]
Closing in favor of https://github.com/vercel/vercel/pull/9360

**Pull request closed without merging**